packages/components/src/InputTime/InputTime.tsx

```diff
@@ -38,50 +45,16 @@ export function InputTime({
 
   function handleBlur() {
     params.onBlur?.();
+    console.log("handleBlur");
 
     // Time inputs doesn't clear the typed value when it's invalid. This should
     // force it to reset the input value when the typed value is invalid.
     if (ref.current) {
+      // when would this ever be invalid? we don't allow any
+      // of the attributes that would make it invalid
       if (!ref.current.checkValidity()) {
         ref.current.value = "";
       }
     }
   }
 }
-
-function dateToTimeString(date?: Date): string {
-  if (!(date instanceof Date)) {
-    return "";
-  }
-
-  // Extract hours and minutes from the Date object
-  const hours = date.getHours().toString().padStart(2, "0");
-  const minutes = date.getMinutes().toString().padStart(2, "0");
-
-  // Return the time string in HH:MM format
-  return `${hours}:${minutes}`;
-}
-
-export function timeStringToDate(timeString: string): Date | undefined {
-  try {
-    const [hours, minutes] = timeString.split(":").map(Number);
-
-    if (
-      isNaN(hours) ||
-      isNaN(minutes) ||
-      hours < 0 ||
-      hours > 24 ||
-      minutes < 0 ||
-      minutes > 60
-    ) {
-      return undefined;
-    }
-
-    const date = new Date();
-    date.setHours(hours, minutes, 0, 0);
-
-    return date;
-  } catch {
-    return undefined;
-  }
-}
```

Comment:
I'm going to try my best to stay on target but I'm very very tempted to reach for `date-fns` rather than doing our own date/string parsing. I've found that library to be really nice for date/time operations, and there's plenty of edge cases to consider when it comes to dates and time.